lib/ansible/modules/cloud/digital_ocean/digital_ocean_droplet_info.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+
+DOCUMENTATION = '''
+---
+module: digital_ocean_droplet_info
+short_description: Gather information about DigitalOcean droplets
+description:
+  - This module can be used to gather information about DigitalOcean droplets.
+version_added: "2.10"
+options:
+  droplet_id:
+    description:
+     - Droplet ID that can be used to identify and reference a droplet.
+    required: false
+    type: int
+  tag_name:
+    description:
+     - Tag name that can be used to filter droplets by a tag.
+    required: false
+    type: str
+author: "Nicolas Boutet (@boutetnico)"
+extends_documentation_fragment: digital_ocean.documentation
+notes:
+  - Version 2 of DigitalOcean API is used.
+requirements:
+  - "python >= 2.6"
+'''
+
+
+EXAMPLES = '''
+- name: Gather information about all droplets
+  digital_ocean_droplet_info:
+    oauth_token: "{{ my_do_key }}"
+
+- name: Gather information about a droplet with given id
+  digital_ocean_droplet_info:
+    oauth_token: "{{ my_do_key }}"
+    droplet_id: 3164444
+
+- name: Gather information about droplets having a given tag
+  digital_ocean_droplet_info:
+    oauth_token: "{{ my_do_key }}"
+    tag_name: "staging"
+'''
+
+
+RETURN = '''
+# Digital Ocean API info https://developers.digitalocean.com/documentation/v2/#list-all-droplets
+data:
+    description: List of droplets on DigitalOcean
+    returned: success and no resource constraint
+    type: dict
```

Comment:
As per your comment, should the type be `list` ?

Reply:
🤦‍♂Nice catch! Fixed.